doc/run_on_minikube.md

```diff
@@ -0,0 +1,113 @@
+# Run PaddleCloud on your local machine
+
+This documentation shows how to run PaddleCloud on minikube.   
+(This surports only mac now.On unbuntu-16.04,we met a bug of kubernetes when we use `hostpath` to volume a file.)
```

Comment:
"Currently, This guide works with Mac OS only. On Ubuntu 16.04, the bug regarding `hostpath` of Kubernetes is a show stopper"
also, do we have a link for this bug in kubernetes repo?

Reply:
https://github.com/kubernetes/kubernetes/issues/44815#issuecomment-297077509
can this help solve the issue?

Reply:
OK, I'' test it later.
Thanks!

Reply:
It's amazing to find that we can volume a file on Linux now!